server/ai_process.go

```diff
@@ -1090,6 +1090,9 @@ func submitLiveVideoToVideo(ctx context.Context, params aiRequestParams, sess *A
 	}
 	clog.V(common.VERBOSE).Infof(ctx, "pub %s sub %s control %s events %s", pub, sub, control, events)
 
+	firstSegmentReceived := make(chan struct{})
```

Comment:
Maybe make this at least size 1 to minimize the chance of late writes blocking after a timeout

Reply:
Ok, updated